website/project/utils.py

```diff
@@ -15,10 +16,7 @@ def serialize_node(*args, **kwargs):
     return _view_project(*args, **kwargs)  # Not recommended practice
 
 PROJECT_QUERY = (
-    # Can encompass accessible projects, registrations, or forks
-    # Note: is_bookmark collection(s) are implicitly assumed to also be collections; that flag intentionally omitted
-    Q('is_deleted', 'eq', False) &
-    Q('type', 'ne', 'osf.collection')
+    DjangoQ(is_deleted=False) & ~DjangoQ(type='osf.collection')
```

Comment:
I would actually make this just Q and change the MODM Q import since it will be removed later.